src/index.js

```diff
@@ -16,6 +16,16 @@ const props = {
 
 const target = document.querySelector('#screen');
 
-usersScreen.render(target, props);
-
-//const router = new Router(routes, document.getElementById('screen'));
+//usersScreen.render(target, props);
+/*usersScreen.update({
```

Comment:
Do not add commented(disable) code to PR without additional comments (descriptions why did you do that).
For me as for reviewer it is not obvious why did you comment (disable) this part.
Example:
```
// Temporary changes for debuge
// TODO: remove comments before merge
//const router = new Router(routes, document.getElementById('screen'));
+//usersScreen.render(target, props);
+/*usersScreen.update({
```